main/src/main/java/tw/firemaples/onscreenocr/floatings/result/ResultView.kt

```diff
@@ -123,6 +123,7 @@ class ResultView(context: Context) : FloatingView(context) {
         btShareOCRText.clickOnce {
             val ocrText = viewModel.ocrText.value ?: return@clickOnce
             Utils.shareText(ocrText)
+            viewResultWindow.visibility = View.GONE
```

Comment:
```suggestion
            onUserDismiss?.invoke()
```

It's a good idea to hide the result window after opening another app.
Maybe we can use `onUserDismiss?.invoke()` to dismiss the highlight at the same time.